src/test/integration/user/userSetInactive.ts

```diff
@@ -1,34 +0,0 @@
-import { userMockTest } from '@test/integration/mock/user'
-
-import { UserStatus } from '@meta/user'
-
-import { UserController } from '@server/controller/user'
-
```

Comment:
why removed ?

Reply:
because there was the following todo in integration.test.ts: // TODO: Remove userInactive / userActive